consumer/pdata/traceid.go

```diff
@@ -43,3 +43,8 @@ func (t TraceID) HexString() string {
 func (t TraceID) IsValid() bool {
 	return data.TraceID(t).IsValid()
 }
+
+// IsEmpty returns true if id doesn't contain at least one non-zero byte.
+func (t TraceID) IsEmpty() bool {
```

Comment:
If we do this, consider to deprecate the other one. We don't need both since they are equivalent.

Reply:
Added a deprecation note for `IsValid()` function.